src/Essentials/src/Essentials.csproj

```diff
@@ -38,6 +38,7 @@
   <ItemGroup Condition=" $(TargetFramework.Contains('-windows')) ">
     <Compile Include="**\*.uwp.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <Compile Include="**\*.uwp.*.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+    <PackageReference Include="Microsoft.Web.WebView2" />
```

Comment:
Why do you need the webview2 here?

Is the webview required to update? Can we do it in a different PR to get better history.

Also, check all projects for references to win2d just in case the is a rogue explicit version.